README.md

```diff
@@ -1,5 +1,12 @@
 # Andromeda
-Andromeda is a website that allows a user to perform dimensional reduction on an uploaded CSV file.
+Andromeda allows users to visualize high dimensional data in a 2-dimensional plot using weighted multidimensional scaling. Through the interactive interface users can explore the relationships within their data by adjusting the column weights and viewing the resulting clusters. Alternatively users can arrange the items into clusters and Andromeda can determine column weights.
```

Comment:
```suggestion
Andromeda allows users to visualize high dimensional data in a 2-dimensional plot using weighted multidimensional scaling. Through the interactive interface users can explore the relationships within their data by adjusting the variable weights and viewing an updated plot.  Alternatively users can arrange the items  in the plot so that Andromeda can determine new variable weights and create a new projection. 
```